themes/default/content/blog/advanced-aws-networking-part-2/index.md

```diff
@@ -0,0 +1,534 @@
+---
+title: "Advanced AWS Networking, Part 2"
+
+# The date represents the post's publish date,
+# and by default corresponds with the date this file was generated.
+# Posts with future dates are visible in development,
+# but excluded from production builds.
+# Use the time and timezone-offset portions of of this value
+# to schedule posts for publishing later.
+date: 2023-04-19
+
+# Use the meta_desc property to provide a brief summary
+# (one or two sentences) of the content of the post,
+# which is useful for targeting search results or social-media previews.
+# This field is required or the build will fail the linter test.
+# Max length is 160 characters.
+meta_desc: Learn how to create a hub-and-spoke architecture in AWS in Python with Pulumi.
+
+# The meta_image appears in social-media previews and on the blog home page.
+# A placeholder image representing the recommended format, dimensions and aspect ratio
+# has been provided for you.
+meta_image: meta.png
+
+# At least one author is required.
+# The values in this list correspond with the `id` properties
+# of the team member files at /data/team/team.
+# Create a file for yourself if you don't already have one.
+authors:
+    - josh-kodroff
+    - andy-taylor
+    - jose-juhala
+
+# At least one tag is required.
+# Lowercase, hyphen-delimited is recommended.
+tags:
+    - aws
+    - networking
+    - hub-and-spoke
+    - python
+
+# See the blogging docs at https://github.com/pulumi/pulumi-hugo/blob/master/BLOGGING.md.
+# for additional details,
+# and please remove these comments before submitting for review.
+---
+
+In this blog series you will learn how to create a hub-and-spoke network architecture in AWS with centralized egress and traffic inspection. In this second installment, we'll show you how to create spoke VPCs to run your workloads, verify centralized egress is working, and then add centralized traffic inspection using Pulumi, the infrastructure as code tool that enables you to manage infrastructure with real programming languages!
+
+<!--more-->
+
+## Introduction
+
+In [AWS Advanced Networking Part 1](https://pulumi.com/blog/advanced-aws-networking-part-1/):
+
+1. Discussed the benefits of a hub-and-spoke network architecture with centralized egress.
+1. Created the shared [AWS Transit Gateway](https://aws.amazon.com/transit-gateway/) resources to be shared among our spoke VPCs and inspection VPC.
+1. Defined our inspection VPC as a Pulumi [component resource](https://www.pulumi.com/docs/intro/concepts/resources/components/).
+
+Now, to finish the architecture, we will need to:
```

Comment:
Just a general note here to try to use "you" (second person) as much as possible (per the Slack conversation about emulating DO's style guidelines) instead of first-person "we"